docs/buildah-bud.md

```diff
@@ -591,6 +592,15 @@ will convert /foo into a `shared` mount point.  The propagation properties of th
 mount can be changed directly. For instance if `/` is the source mount for
 `/foo`, then use `mount --make-shared /` to convert `/` into a `shared` mount.
 
+If you use the special `overlay=` options, the directory on the host will be
+mounted as an overlay into the container.  When the container stops running,
+any contents written to the volume will be destroyed.  This allows `RUN`
+command containers to share a directory from the host and even modify it.  When
+the container completes the content will be removed from the host and any
```

Comment:
I'm possibly confused, will the content be removed from the host or the container?  I think container.  You say in the second part of the sentence that subsequent "RUN" will get the content from the host....